relay-server/src/actors/project_redis.rs

```diff
@@ -38,6 +39,20 @@ impl From<serde_json::Error> for RedisProjectError {
     }
 }
 
+fn parse_redis_response(raw_response: &[u8]) -> Result<ProjectState, RedisProjectError> {
+    let decoded = metric!(timer(RelayTimers::ProjectStateDecompression), {
+        zstd::decode_all(raw_response)
+    });
+
+    let raw_response: Cow<[u8]> = match decoded {
+        Ok(decoded) => decoded.into(),
```

Comment:
can't you just borrow here to avoid the Cow?

Reply:
Yep, thanks.